src/middlewared/middlewared/plugins/failover_/disabled_reasons.py

```diff
@@ -160,6 +161,16 @@ def get_remote_reasons(self, app, ifaces, reasons):
             mismatch_disks = self.middleware.call_sync('failover.mismatch_disks')
             if mismatch_disks['missing_local'] or mismatch_disks['missing_remote']:
                 reasons.add(DisabledReasonsEnum.MISMATCH_DISKS.name)
+
+            local_nics = self.middleware.call_sync('interface.get_nics')
+            try:
+                remote_nics = self.middleware.call_sync('failover.call_remote', 'interface.get_nics')
```

Comment:
IIRC these will be converted to lists during the API calls, and I **don't** think we are _guaranteed_ ordering.

Might be safer to make them sets again locally before doing the comparison.  e.g.
```
local_nics = set(self.middleware.call_sync('interface.get_nics'))
```

Reply:
Mea culpa.